src/tests/miscellaneous/mock-axe-android/AccessibilityInsights/config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Should the folder name and `package.json` script names still referencing "axe-android" also be updated